src/script/campaign/mod/vco-listeners.lua

```diff
@@ -2,6 +2,29 @@ local vco = core:get_static_object("vco");
 
 -- COMMON --
 
+-- POOLED RESOURCES --
+
+local function update_faction_pooled_resource_earnings(faction, pooled_resource_key, amount)
```

Comment:
Check identation for this function in general.

Reply:
Also, I know I am being picky with this one, but `faction` to `faction_name`, so we know it is not an object, and `amount` shoud maybe `difference`, if it can be both positive and negative. Second pat is not a real suggestion tho, just a thought.

Reply:
`amount` is the key used in the interface so I'd prefer to stick with that. I don't think it confers positive/negative.

Indentation will be fixed.